pvlib/test/test_singlediode_methods.py

```diff
@@ -125,3 +126,135 @@ def test_brentq_fs_495():
                                 method='lambertw')
     assert np.isclose(pvs_ixx, ixx)
     return isc, voc, imp, vmp, pmp, i, v, pvs
+
+
+def pvsyst_fs_495():
+    """
+    PVSyst First Solar FS-495 parameters.
+
+    Returns
+    -------
+    dictionary of PVSyst First Solar FS-495 parameters
+    """
+    fs_495 = dict(d2mutau=1.31, alpha_sc=0.00039, gamma_ref=1.48,
+                  mu_gamma=0.001, I_o_ref=0.962e-9, R_sh_ref=5000,
+                  R_sh_0=12500, R_sh_exp=3.1, R_s=4.6, beta_oc=-0.2116,
+                  EgRef=1.475, cells_in_series=108, cells_in_parallel=2,
+                  I_sc_ref=1.55, V_oc_ref=86.5, I_mp_ref=1.4,
+                  V_mp_ref=67.85,
+                  temp_ref=25, irrad_ref=1000)
+    Vt = 0.025693001600485238  # thermal voltage at reference (V)
+    nNsVt = fs_495['cells_in_series'] * fs_495['gamma_ref'] * Vt
+    Vd = fs_495['I_sc_ref'] * fs_495['R_s']  # diode voltage at short circuit
+    Id = fs_495['I_o_ref'] * (np.exp(Vd / nNsVt) - 1)  # diode current (A)
+    Ish = Vd / fs_495['R_sh_ref']  # shunt current (A)
+    # builtin potential difference (V)
+    dv = VOLTAGE_BUILTIN * fs_495['cells_in_series'] - Vd
+    # calculate photo-generated current at reference condition (A)
+    fs_495['I_L_ref'] = (
+        (fs_495['I_sc_ref'] + Id + Ish) / (1 - fs_495['d2mutau'] / dv)
+    )
+    return fs_495
+
+
+PVSYST_FS_495 = pvsyst_fs_495()  # PVSyst First Solar FS-495 parameters
+
+
+def test_pvsyst_fs_495_recombination_loss():
```

Comment:
I think the test is more general than just the `fs_495` module - I'd drop that nomenclature from the function name.

Reply:
✔️ agreed, changed to `test_pvsyst_recombination_loss`